web-api/src/business/useCases/courtIssuedDocument/fileAndServeCourtIssuedDocumentInteractor.deadlines.test.ts

```diff
@@ -0,0 +1,255 @@
+      docketNumber: MOCK_CASE.docketNumber,
+      documentTitle: 'Order',
+      documentType: 'Order',
+      eventCode: 'O',
+      signedAt: '2019-03-01T21:40:46.415Z',
+      signedByUserId: docketClerkUser.userId,
+      signedJudgeName: 'Dredd',
+      userId: docketClerkUser.userId,
+      workItem: mockWorkItem,
+    };
+
+    caseRecord = {
+      ...MOCK_CASE,
+      associatedJudge: judgeUser.name,
+      docketEntries: [
+        mockDocketEntryWithWorkItem,
+        {
+          docketEntryId: 'c54ba5a9-b37b-479d-9201-067ec6e335bc',
+          docketNumber: MOCK_CASE.docketNumber,
+          documentTitle: 'Order to Show Cause',
+          documentType: 'Order to Show Cause',
+          eventCode: 'OSC',
+          signedAt: '2019-03-01T21:40:46.415Z',
+          signedByUserId: docketClerkUser.userId,
+          signedJudgeName: 'Dredd',
+          userId: docketClerkUser.userId,
+        },
+        {
+          docketEntryId: '7f61161c-ede8-43ba-8fab-69e15d057012',
+          docketNumber: MOCK_CASE.docketNumber,
+          documentTitle: 'Transcript of [anything] on [date]',
+          documentType: 'Transcript',
+          eventCode: TRANSCRIPT_EVENT_CODE,
+          userId: docketClerkUser.userId,
+        },
+      ],
+    };
+
+    mockDeadline = {
+      caseDeadlineId: 'test-deadline-id',
+      deadlineDate: '2030-01-20T00:00:00.000Z',
+      description: 'Test Deadline',
+    };
+
+    consolidatedCaseExample = {
+      petitioners: [] as TPetitioner[],
+      caseCaption: '',
+      caseType: 'Deficiency' as CaseType,
+      createdAt: '',
+      partyType: '',
+      petitionPaymentStatus: '',
+      procedureType: '',
+      receivedAt: '',
+      status: '' as CaseStatus,
+      sortableDocketNumber: 0,
+      docketEntries: [],
+      hearings: [],
+      correspondence: [],
+      consolidatedCases: [],
+      docketNumberWithSuffix: '',
+    };
+
+    getUserById.mockResolvedValue(docketClerkUser as DbUser);
+
+    getCaseByDocketNumber.mockResolvedValue(caseRecord);
+    getCasesByDocketNumbers.mockResolvedValue([caseRecord]);
+
+    fileAndServeDocumentOnOneCase.mockImplementation(
+      ({ caseEntity }) => caseEntity,
+    );
+
+    applicationContext
+      .getUseCaseHelpers()
+      .serveDocumentAndGetPaperServicePdf.mockReturnValue({
+        pdfUrl: mockPdfUrl,
+      });
+
+    applicationContext
+      .getUseCaseHelpers()
+      .stampDocumentForService.mockReturnValue(testPdfDoc);
+
+    applicationContext.getUseCaseHelpers().autoGenerateDeadline = jest
+      .fn()
+      .mockResolvedValue([mockDeadline]);
+  });
+  it('should create a deadline if the Status report order has a due date and order type', async () => {
+    const mockAutoGeneratedDeadlineDocketEntry = {
+      ...mockDocketEntryWithWorkItem,
+      docketEntryId: 'ed68e80a-cab3-44b9-9280-03816bba8cba',
+      eventCode: 'O',
+    };
+    caseRecord.docketEntries.push(mockAutoGeneratedDeadlineDocketEntry);
```

Comment:
Just a note that updating the `docketEntries` array in one test could make other tests more fragile. Any reason we don't just add this docket entry to the `caseRecord` initialization?